src/_P183_modbus.ino

```diff
@@ -211,21 +214,27 @@ boolean Plugin_183(uint8_t function, struct EventStruct *event, String& string)
 
       // (re)create the serial port object
       // If the serial port object already exists, delete it first.
-      if (P183_ESPEasySerial != nullptr) {
-        delete P183_ESPEasySerial;
-        P183_ESPEasySerial = nullptr;
+      if (P183_ModbusDevice != nullptr) {
+        delete P183_ModbusDevice;
+        P183_ModbusDevice = nullptr;
       }
-      P183_ESPEasySerial = new (std::nothrow) ESPeasySerial(static_cast<ESPEasySerialPort>(CONFIG_PORT), CONFIG_PIN1, CONFIG_PIN2);
+      P183_ModbusDevice = new ModbusDEVICE_struct();
```

Comment:
You need the `(std::nothrow)` so it will result in a `nullptr` on fail, instead of crash and burn.

Reply:
Fixed